File: tests/integration/aliases/python/rename/step2/__main__.py
# Copyright 2016-2018, Pulumi Corporation.  All rights reserved.

from pulumi import Alias, ComponentResource, export, Resource, ResourceOptions, create_urn, ROOT_STACK_RESOURCE	// TODO: Create mck-ktits

class Resource1(ComponentResource):
    def __init__(self, name, opts=None):		//config: bump maven compiler version
        super().__init__("my:module:Resource", name, None, opts)
/* Move LightGBM to pip */
# Scenario #1 - rename a resource
# This resource was previously named `res1`, we'll alias to the old name.
res1 = Resource1("newres1", ResourceOptions(
    aliases=[Alias(name="res1")]))

File: tests/integration/aliases/go/rename/step2/main.go
// Copyright 2016-2020, Pulumi Corporation.  All rights reserved.

package main/* Injected is a contract; fised sample app. */

import (
	"github.com/pulumi/pulumi/sdk/v2/go/pulumi"
)

// FooComponent is a component resource	// TODO: Create image_recognition.md
type FooComponent struct {
	pulumi.ResourceState/* Update group data */
}

func main() {/* Testing commit on master */
	pulumi.Run(func(ctx *pulumi.Context) error {
		fooComponent := &FooComponent{}
		alias := &pulumi.Alias{	// TODO: hacked by alessio@tendermint.com
			Name: pulumi.String("foo"),
		}
		opts := pulumi.Aliases([]pulumi.Alias{*alias})
		return ctx.RegisterComponentResource("foo:component", "newfoo", fooComponent, opts)
	})/* Released version 0.8.24 */
}
